From 76b86cba72755ca261ad8ef8155145c97b2a57d1 Mon Sep 17 00:00:00 2001
From: Rob Clark <rclark@redhat.com>
Date: Wed, 17 Dec 2014 21:38:45 -0500
Subject: [drm] radeon: Add missing lines to ci_set_thermal_temperature_range

Message-id: <1418852380-13061-35-git-send-email-rclark@redhat.com>
Patchwork-id: 102165
O-Subject: [RHEL7 drm 34/89] drm/radeon: Add missing lines to ci_set_thermal_temperature_range
Bugzilla: 1173317
RH-Acked-by: Jerome Glisse <jglisse@redhat.com>
RH-Acked-by: Dave Airlie <airlied@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1173317
Upstream: since v3.16.4

commit ac3e0959ede8f70c44eb936c7ea9b83fb41b2826

Author:     Oleg Chernovskiy <algonkvel@gmail.com>
AuthorDate: Mon Aug 11 21:53:46 2014 +0400
Commit:     Rob Clark <rclark@redhat.com>
CommitDate: Wed Dec 10 17:19:42 2014 -0500

    drm/radeon: Add missing lines to ci_set_thermal_temperature_range

    commit 6bce8d9772c1c606921a9c99e566eb14202f6669 upstream.

    Properly set the thermal min and max temp on CI.
    Otherwise, we end up setting the thermal ranges
    to 0 on resume and end up in the lowest power state.

    Signed-off-by: Oleg Chernovskiy <algonkvel@gmail.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Rob Clark <rclark@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/radeon/ci_dpm.c b/drivers/gpu/drm/radeon/ci_dpm.c
index 022561e..d416bb2 100644
--- a/drivers/gpu/drm/radeon/ci_dpm.c
+++ b/drivers/gpu/drm/radeon/ci_dpm.c
@@ -869,6 +869,9 @@ static int ci_set_thermal_temperature_range(struct radeon_device *rdev,
  WREG32_SMC(CG_THERMAL_CTRL, tmp);
 #endif
 
+ rdev->pm.dpm.thermal.min_temp = low_temp;
+ rdev->pm.dpm.thermal.max_temp = high_temp;
+
  return 0;
 }
 
-- 
1.7.1